assets/css/push.css

```diff
@@ -177,55 +175,43 @@
 }
 
 #wpadminbar #distributor-push-wrapper button {
-	background: #0085ba;
-	border-color: #0073aa #006799 #006799;
-	border-width: 1px;
-    border-style: solid;
-	box-shadow: 0 1px 0 #006799;
-	color: #fff;
-	text-decoration: none;
-	text-shadow: 0 -1px 1px #006799, 1px 0 1px #006799, 0 1px 1px #006799, -1px 0 1px #006799;
-	height: 30px;
-	line-height: 28px;
-	padding: 0 12px 2px;
-	border-radius: 3px;
```

Comment:
we should keep the `border-radius` and `text-shadow` because they're overridden by admin bar style.